code/modules/vehicles/vehicle.dm

```diff
@@ -221,6 +223,9 @@
 /obj/vehicle/proc/RunOver(mob/living/carbon/human/H)
 	return		//write specifics for different vehicles
 
+/obj/vehicle/proc/add_slowdown(slowdown_type, slowdown_value)
+
```

Comment:
???

Reply:
oops, i was experimenting and forgot to remove